src/components/exchange/ExchangeAssetList.tsx

```diff
@@ -75,23 +79,13 @@ const HeaderTitleGradient = styled(GradientText).attrs({
   weight: 'heavy',
 })({});
 
-const HeaderTitleWrapper = styled.View({});
-
 const contentContainerStyle = { paddingBottom: 9.5 };
 const scrollIndicatorInsets = { bottom: 24 };
-const keyExtractor = ({ uniqueId }) => `ExchangeAssetList-${uniqueId}`;
-
-const getItemLayout = ({ showBalance }, index) => {
```

Comment:
I removed this because it wasnt actually implemented correctly and so at best was being ignored or at worst having to recalculate anyway.